packages/node_modules/samples/browser-plugin-meetings/app.js

```diff
@@ -258,6 +266,13 @@ function getMediaSettings() {
 
   toggleSourcesMediaDirection.forEach((options) => {
     settings[options.value] = options.checked;
+
+    if (options.sendShare && (isSafari || isiOS)) {
+      // It's been observed that trying to setup a Screenshare at join along with the regular A/V streams
+      // causes Safari to loose track of it's user gesture event due to getUserMedia & getDisplayMedia being called at the same time (through our internal setup)
+      // It is recommended to join a meeting with A/V streams first and then call `meeting.shareScreen()` after joining the meeting successfully (on all browsers)
+      settings[options.value] = false;
```

Comment:
Since this is the sample app too, can you log this out to the console too?
```js
console.log('MeetingControsl#getMediaSettings() :: Please call `meeting.shareScreen()` after joining the meeting');

Reply:
I thought we used —> instead of ::

Reply:
That's for logs, in the sample app the standard seems to be `::`